OpenRA.Game/Map/Map.cs

```diff
@@ -494,7 +489,7 @@ public Ruleset PreloadRules()
 			return rules.Value;
 		}
 
-		public void Save(string toPath)
+		public void Save(IReadWritePackage toContainer)
 		{
 			MapFormat = 8;
```

Comment:
Not related to this PR at all, but could we make this `MapFormat = SupportedMapFormat`, please?